src/ui/voting/useVotingPowerForProtocol.tsx

```diff
@@ -61,7 +75,8 @@ function useVestingVaultVotingPower(): string {
 
     const vestingVaultVotingPower = formatEther(vestingVaultVotingPowerBN);
     return vestingVaultVotingPower;
-    // events are immutable, only update if there are new events.
```

Comment:
This seems like a nice comment to leave around. It explains why we're doing array?.length in the deps array, when the hook itself actually uses the list.

Reply:
oops thanks, yeah that should still be there